express-notes-api/index.js

```diff
@@ -0,0 +1,97 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+const data = require('./data.json');
+const notes = data.notes;
+const jsonMiddleWare = express.json();
+
+app.get('/api/notes', (req, res) => {
+  const array = [];
+  for (const property in notes) {
+    array.push(notes[property]);
+  }
+  res.status(200).json(array);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const id = Number(req.params.id);
+  if (!id || id < 0) {
+    res.status(400).json({ error: 'id must be a positive integer' });
```

Comment:
Positive numbers with decimals should also produce a `400 Bad Request`:

---

![image](https://user-images.githubusercontent.com/16143771/144654514-5def090e-b963-447c-a4ff-bc22fc359e49.png)